libs/utils/udi-service.ts

```diff
@@ -0,0 +1,3 @@
+export function buildUdi(entityType: string, guid: string) {
+	return 'umb://' + entityType + '/' + guid.replace(/-/g, '');
```

Comment:
Consider using template literals for readability

```suggestion
	return `umb://${entityType}/${guid.replace(/-/g, '')}`;
```